docs/moleculenet.rst

```diff
@@ -197,3 +201,13 @@ UV Datasets
 -----------
 
 .. autofunction:: deepchem.molnet.load_uv
+
+
+.. _`datasets already available in MolNet`: http://moleculenet.ai/datasets-1
+.. _`Contribution guidelines`: https://github.com/deepchem/deepchem/blob/master/CONTRIBUTING.md
+.. _`issue`: https://github.com/deepchem/deepchem/issues
+.. _`deepchem.molnet.load_function`: https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/load_function
+.. _`deepchem.molnet.load_function.load_dataset_template`: https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/load_function/load_dataset_template.py
+.. _`deepchem.molnet.defaults`: https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/defaults.py
+.. _`deepchem.molnet.__init__.py`: https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/__init__.py
+.. _`template`: https://github.com/deepchem/deepchem/blob/master//molnet_pr_template.md
```

Comment:
I think there's a typo here? Should be https://github.com/deepchem/deepchem/blob/master/.github/PULL_REQUEST_TEMPLATE/molnet_pr_template.md

Reply:
I fixed!